ubtue_Brill.js

```diff
@@ -67,7 +67,27 @@ function getSearchResults(doc) {
 	return found ? items : false;
 }
 
+// extract author names form Open Graph Metadata, if dot is included in names, e.g. "Th. Emil Homerin" https://brill.com/view/journals/jss/9/1/article-p52_3.xml 
+// because in this case the first string of the author name with dot will be removed by the system.
+function extractAuthors(doc) {
+	let authorsElement = ZU.xpathText(doc, '//meta[@property="article:author"]//@content');
+	if (authorsElement.match(/\./))
+		return authorsElement ? authorsElement.split(',') : '';
+	return false;
+}
+
 function postProcess(doc, item) {
+	if (extractAuthors(doc)) {
+		item.creators = [];
+		for (let author of extractAuthors(doc)) {
+			author = author.split(' ');
+			item.creators.push({			
+			lastName: author.pop().replace('†', ''),
+			firstName: author.join().replace(',', ' '),
+			creatorType: 'author'
+			});
+		}
+	}
```

Comment:
This seems overly comlex: In `extractAuthors` the string is split by comma. Then there is another splitting by space ,  one value removed from the array and then joined and replaced ',' again afterwards. Given that this is now the generic approach for _all_ Brill authors chances are high that not all cases are handled correctly.

Reply:
this is not generic approach, because it is applied when this condition is met`if (authorsElement.match(/\./))`

Reply:
Sorry, did not look closely enough. But apart from appearing fragile (do all names that contain a dot anywhere really imply the  structure presupposed later?) this should be in some way obvious from the code i.e. by using a predicate `OpenGraphMetadataNameContainsDot`. And currently the return value of `extractAuthor` currently can have three types (Array, string, bool) , this should be unified. But still, I would strongly discourage from adding many special treatments for a potentially very limited number of cases.